G1ANT.Addon.Net/SimplifiedMessageSummary.cs

```
@@ -50,6 +52,9 @@ public List<object> Attachments
             get => CreateAttachmentStructuresFromAttachments(FullMessage.Attachments);
             set
             {
+                if (value == null)
+                    return;
```

Consider throwing `NAE` here instead of silently ignoring null value.